build.gradle.kts

```diff
@@ -160,7 +160,7 @@ intellijPlatform {
       } else {
         ide(IntelliJPlatformType.IntellijIdeaCommunity, ideaVersion)
       }
-      recommended()
+//      recommended()
```

Comment:
intentional?

Reply:
Yes, thank for pointing this out.  With this early version of the Meerkat Canary, this query yields in an exception in the verify cli command.